temporalcli/commands.activity.go

```diff
@@ -0,0 +1,73 @@
+package temporalcli
+
+import (
+	"fmt"
+
+	"go.temporal.io/api/common/v1"
+	"go.temporal.io/api/failure/v1"
+	"go.temporal.io/api/workflowservice/v1"
+)
+
+func (c *TemporalActivityCompleteCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	metadata := map[string][]byte{"encoding": []byte("json/plain")}
+	resultPayloads, err := CreatePayloads([][]byte{[]byte(c.Result)}, metadata, false)
+	if err != nil {
+		return err
+	}
+
+	_, err = cl.WorkflowService().RespondActivityTaskCompletedById(cctx, &workflowservice.RespondActivityTaskCompletedByIdRequest{
```

Comment:
While this is the approach of the old SDK, `cl.CompleteActivityByID` would have worked here too (you'd wrap the payload in `rawValue`). Arguably it'd be cleaner/clearer to use the high-level SDK client, but don't need to change now, but we might consider using the high-level API in the future.

Reply:
OK I'm happy to make that change and it would be easy, however, it looks like it doesn't support `identity` so I'm going to leave it for now.